docs_src/getting-started/Ch-GettingStartedSDK-Go.md

```diff
@@ -264,6 +264,67 @@ See the [Device MQTT Service](https://github.com/edgexfoundry/device-mqtt-go/tre
 - [See here for custom section on the configuration.toml file](https://github.com/edgexfoundry/device-mqtt-go/blob/v2.0.0/cmd/res/configuration.toml#L86-L108)
 - [See here for loading, validating and watching the configuration](https://github.com/edgexfoundry/device-mqtt-go/blob/v2.0.0/internal/driver/driver.go#L54-L69)
 
+## Custom Device Service Metrics
+
+!!! edgey "EdgeX 2.3"
+    Custom Device Service Metrics are new in EdgeX 2.3, but currently only for the **Go SDK**
+
+The Custom Device Service Metrics capability allows for custom device services to define, collect and report their own custom service metrics.
```

Comment:
"custom device services" - what wouldn't be a "custom" here? Just device-virtual?
I think you can drop the "custom" word before device services.

Reply:
I consider custom device service ones written outside the EdgeX community, i.e. not any of ours. But technically event ours are "custom" and can add additional metrics beyond that supplied by the SDK.

Reply:
Will remove word "custom"